recipe/build.sh

```diff
@@ -12,11 +12,45 @@ then
 fi
 
 ./configure \
-        --prefix="${PREFIX}" \
-        --enable-shared \
-        --enable-pic \
-        --enable-libx264 \
-        --disable-doc \
-        --enable-gpl
+       --prefix="${PREFIX}" \
+       --disable-doc \
+       --disable-debug \
+       --disable-static \
+       --disable-stripping \
+       --enable-avisynth \
+       --enable-avresample \
+       --enable-fontconfig \
+       --enable-gpl \
```

Comment:
We might be in conflict with some things here. GPLing the binary makes some options illegal. Normally, `ffmpeg` will just not build in that case. Can't remember which options though.